src/gui/tray/talkreply.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include <QtCore>
+#include <QPointer>
+
+class QJsonArray;
+
+namespace OCC {
+class Activity;
+class JsonApiJob;
+class AccountState;
+
+class TalkReply : public QObject
+{
+    Q_OBJECT
+
+public:
+    explicit TalkReply(AccountState *accountState, QObject *parent = nullptr);
+
+    void sendReplyMessage(const QString &conversationToken, const QString &message, const QString &replyTo = {});
+
+signals:
+    void replyMessageSent(const QString &message);
+
+private:
+    AccountState *_accountState;
```

Comment:
I would maybe add a ` = nullptr` to ensure it is always initialized (even if later, we would add another constructor with different behavior)